src/main/java/cd/go/artifact/docker/registry/Constants.java

```diff
@@ -25,7 +25,7 @@
     String EXTENSION_TYPE = "artifact";
 
     // The extension point API version that this plugin understands
-    String API_VERSION = "1.0";
+    String API_VERSION = "2.0";
```

Comment:
This will break requests from plugin to the server. e.g -

https://github.com/gocd/docker-registry-artifact-plugin/blob/022e1bb8ad7468f71d351f60b26f03f92a7d405a/src/main/java/cd/go/artifact/docker/registry/ConsoleLogger.java#L46

Ideally, it should be like -

https://github.com/gocd-contrib/elastic-agent-skeleton-plugin/blob/master/src/main/java/com/example/elasticagent/Constants.java#L31-L34

Please check where `API_VERSION` is used and create a separate constants as mentioned above.

Reply:
@bdpiparva I've fixed this in the latest commit. Can you take a look?